### DISCUSSION TITLE
Suggest that obsidian notes can be seen in zotero as well

### DISCUSSION THREAD

**Discussion by SunProg** [aidenlx/obsidian-zotero#234] [Ideas]
Personally, I barely find literature notes itself. I just skim through the literature and refer the notes in the zotero app.

However, it is not possible for now to check the notes in the zotero, which is inconvenient especially on mobile.

I think it might be good if I can check the literature notes in the zotero app as well!

**Comment by siainbuletin** [comment]
It would indeed be pretty cool but I feel it might clutter up the Zotero library. Maybe it could be an icon placed next to the name of the parent item that would signal the existence of a literature note in Obsidian (like the Zotero Tags plugin does with the colored blocks next to the name) and then link back to it

**Comment by aidenlx** [comment]
Part of this landed in v2. The Zotero companion registers an **Obsidian Note** column in the item list, so you can see at a glance which items already have a literature note, and the item right-click menu has **Open Literature Note in Obsidian** to jump straight there. That is close to the indicator-next-to-the-item idea raised in the comment above.

What it does not do is render note contents inside Zotero — the column is a status indicator plus a jump, not a viewer. Closing on that basis; if reading note bodies inside Zotero (especially on mobile) is the part you actually want, that deserves its own thread.

v2 beta is available now: https://zotlit.aidenlx.site/docs/install-zotlit
Migration guide: https://zotlit.aidenlx.site/docs/how-to/migrate-from-v1
